dashboard/pkg/epinio/dialog/ExportAppDialog.vue

```diff
@@ -25,12 +32,18 @@ export default {
         title:       this.t('promptRemove.title'),
         applyMode:   'export',
         applyAction: this.exportApplicationManifest,
-      }
+      },
+      zipParts:        this.resources[0].applicationParts.filter((part) => part !== APPLICATION_PARTS.MANIFEST),
+      showProgressBar: false,
+      downloadingPart: null,
+      progressBar:     0,
+      colorStops:      { 0: '--success', 100: '--success' },
```

Comment:
not sure i picked up on this before, these should be primary colour.

Reply:
`primary` is gray in the `PercentageBar`.

Reply:
the issue is that the bar is green when in progress, when it should be the primary colour. once complete we could change it to success / green, though it might be jarring as it'll flash up on screen. if the progress bar shows `grey` for `--primary` we should find another colour to show

Reply:
I'll try to change the PercentageBar component